diff --git pylint/test/input/func_3k_removed_stuff_py_30.py pylint/test/input/func_3k_removed_stuff_py_30.py
--- pylint/test/input/func_3k_removed_stuff_py_30.py
+++ pylint/test/input/func_3k_removed_stuff_py_30.py
@@ -1,7 +1,7 @@
+from __future__ import generators, print_function
 """test relative import"""
 # pylint: disable=no-absolute-import
 __revision__ = filter(None, map(str, (1, 2, 3)))
-from __future__ import generators, print_function
 
 import func_w0302
 
